.idea/misc.xml

```diff
@@ -1,6 +0,0 @@
-<project version="4">
```

Comment:
this will be recreated as soon as anyone tries to run one of the python binaries

Reply:
Ah maybe need to add it to the gitignore then

Reply:
Adding it to the gitignore means that if we _do_ make a change it will not be kept in source control...

Reply:
Do we want to make changes to this file though? As long as IntelliJ just uses some python on their computer we don't really care for which right?

Reply:
I don't know what else goes in misc.xml though. We can remove it and find out...